arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -2667,6 +2667,10 @@
           "type": "boolean",
           "description": "Gets or sets whether the extension handler should be automatically upgraded across minor versions."
         },
+        "forceUpdateTag": {
+          "type": "string",
+          "description": "Gets or sets whether the extension handler should be forced to re-run even if the extension configuration has not changed."
```

Comment:
@markcowl  Seems like maybe we should have a different description for GET in Swagger? Since `"Gets or sets whether the extension handler should be forced to re-run even if the extension configuration has not changed."` should ideally be `Specifies whether the extension configuration was forced to rerun` in case of a GET call